Sources/Amplitude/Amplitude.m

```diff
@@ -811,7 +806,7 @@ - (void)uploadEventsWithLimit:(int)limit {
             return;
         }
 
-        [self makeEventUploadPostRequest:self->_serverUrl events:eventsString numEvents:numEvents maxEventId:maxEventId maxIdentifyId:maxIdentifyId];
+        [self makeEventUploadPostRequest:self->_serverUrl events:eventsString numEvents:numEvents maxEventId:numEvents maxIdentifyId:numIdentify];
```

Comment:
I didn't change the original function signature just in case people use the function on their end. Even this function is not in the header file, there is no public/private file in objective-c, so the "private" function can also be used outside. But feels bad about the no meaningful name. Feel free to share your opinion.

Reply:
All variables and functions in .m file is not accessible except they use objc_send_msg. So it's ok to change the signature.